web-app/src/games/games.service.ts

```diff
@@ -13,11 +13,10 @@ import {
 import { Coordinates, deepCloneObject } from '../common/utils';
 import { UsersService } from '../users/users.service';
 
-export interface GameAndStatus {
+export interface GameAndDisplayStatus {
   game: Game;
   readyToPlay: boolean;
```

Comment:
This "readyToPlay" property is now redundant with the status defined inside the Game entity !

Reply:
Indeed we should remove this attr and keep only `Game.status`. This would have impacts on both the web-app and the mobile-app, so I created this issue instead: https://github.com/marmelab/Hex/issues/68